requirements/pip-tools.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.8
+# This file is autogenerated by pip-compile with Python 3.10
```

Comment:
Do you know why this is using python 3.10?

Reply:
I run this in my local environment